### PR TITLE
Use libhandy for rounded window corners

### PR DESCRIPTION
**Pull request by jpnurmi**
The old code for GtkHeaderBar in ubuntu_wizard is kept for WSL.

Building a snap requires:
- #611
- #617

Close: #583

### CODE REVIEW

File: packages/ubuntu_wizard/linux/ubuntu_wizard_plugin.cc [hunk 1]
```diff
@@ -16,6 +19,57 @@ struct _UbuntuWizardPlugin {
...
 G_DEFINE_TYPE(UbuntuWizardPlugin, ubuntu_wizard_plugin, g_object_get_type())
 
+static GtkWidget* get_header_bar(GtkWindow* window) {
+#ifdef HAVE_LIBHANDY
+  if (HDY_IS_WINDOW(window) || HDY_IS_APPLICATION_WINDOW(window)) {
+    GtkWidget* deck = gtk_bin_get_child(GTK_BIN(window));
+    GList* content = gtk_container_get_children(GTK_CONTAINER(deck));
+    GList* children = gtk_container_get_children(GTK_CONTAINER(content->data));
+    GtkWidget* header_bar = GTK_WIDGET(children->data);
+    g_list_free(content);
```
Comment (robert-ancell) [inline packages/ubuntu_wizard/linux/ubuntu_wizard_plugin.cc:29]: You can use `g_autoptr(GList)` to avoid these lines.

**Review by robert-ancell — APPROVED** [review]
LGTM!

**Comment by jpnurmi** [comment]
I forgot to attach a screenshot:
Screenshot from 2022-01-19 10-50-27

**Comment by Feichtmeier** [comment]
@jpnurmi awesome! Thank you very much :clinking_glasses:

**Pull request merged**